tests/PHPCR/Tests/Util/Console/Command/NodesUpdateCommandTest.php

```diff
@@ -94,6 +94,7 @@ public function testNodeUpdate($options)
         $this->setupQueryManager($options);
 
         $args = [
+            '--query-language' => 'jcr-sql2',
```

Comment:
the original solution was to completely drop this argument: https://github.com/wachterjohannes/phpcr-utils/commit/b222c2ad9f4772257106c77da8048f9ced5af111 . for the 1.2 branch i think it was simply never fixed, i think we can just leave it as is there. but can you please delete it here?

the thing is: sql2 is default, but if you specify a query language option, you have to specify a value for it.

Reply:
@dbu should be fixed